### ISSUE TITLE
Candidate page UI issues/improvements

### ISSUE THREAD

**Issue by ejanzer** [codeforsanjose/open-disclosure#137]
- SideNav container padding causes H3s to be misaligned (the candidate's name should be aligned with the section headers beneath)
- The 'See all contributions' text doesn't align properly with the arrow, doesn't match arrow color, and also doesn't do anything when you click on it (where should it go?)
- Voter's Edge link doesn't go anywhere, that needs to be added to candidate JSON file
- Profile photo link needs to be added to candidate JSON file
- Candidate description/bio (what should go there?) needs to be added to candidate JSON file
- Fix positioning/wrap of bio links (row or col?)
- We currently combine RCPT and LOAN to calculate total contributions. Do we need some UX treatment to indicate this?
- We have a lot of expenditure data, so I'm currently only showing top 4 categories. Should we make this expandable?
- Need more padding between sections
- Text is wrapping too much in the chart labels, can we adjust the width/margin for the label to make it.. not do that?
- Breakdown by region is currently missing the "In San José" category - need to get that from the API (when we do, we need to decide - should "In California" also include SJ, or no?)
- What should the office title be - 'District 4', 'District 4 Representative', 'Council Member District 4'? We currently have all 3 in different places, I think.

**Comment by ejanzer** [comment]
@alessandro-pianetta: Did you make note of the UX issues on the candidate page we talked about with Diane? I tried to just jot down everything I saw looking at the page just now, but I think I'm missing some stuff.

**Comment by alessandro-pianetta** [comment]
@ejanzer Yes, the notes I took ended up being kind of bad but I got most of them in #135 and #136. I might have missed jotting one or two of those down though

**Comment by ejanzer** [comment]
Oh great, I should have checked out your PRs first! Yeah some of these are new as I'm working on hooking up the candidate data, I just wanted to make a note of them someplace as I thought of them.

**Comment by alessandro-pianetta** [comment]
For F I think we decided to render them as a column due to wrapping issues at smaller screen sizes between 1440 and the break point.
